src/ChannelServer/Network/Sending/Helpers/PartyHelper.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Aura.Channel.World.Entities;
+using Aura.Mabi.Network;
+using Aura.Channel.World;
+
+namespace Aura.Channel.Network.Sending.Helpers
+{
+    public static class PartyHelper
+    {
+
+        public static void SettingsParse(Party party, Packet packet)
+        {
+            var type = (PartyType)packet.GetInt();
+            var name = packet.GetString();
+
+            if (type == PartyType.Dungeon)
+            {
+                var dungeonLevel = packet.GetString();
+                var info = packet.GetString();
+
+                party.SetDungeonLevel(dungeonLevel);
+                party.SetInfo(info);
+            }
+            var password = packet.GetString();
+            var maxSize = packet.GetInt();
+
+            party.SetType(type);
+            party.SetName(name);
+            party.SetPassword(password);
+            party.SetSize(maxSize);
+
+            Send.PartySettingUpdate(party);
+
+            if (party.IsOpen)
+                Send.PartyMemberWantedRefresh(party);
+        }
+
+        /// <summary>
+        /// Constructs the party info packet, because this is used in a number of packets.
+        /// </summary>
+        /// <param name="party"></param>
+        /// <param name="packet"></param>
+        public static void BuildPartyInfo(Party party, ref Packet packet)
```

Comment:
My bad... I actually missed they were in helpers xD" But! Parsing should go into handling, into the actual handler, and helpers should use `Foobar(this Packet packet, ...)`, so you can use them like this: `packet.Foobar(...)`.